README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# CSS Homework1
+
+> ## Fernando Alonso Regalado
+
+![Bio Picture](https://github.com/fernandoregalado/hw01/blob/master/picture_bio.jpg)
+
+Hi! I am a second year **MPP student** :neckbeard: and peruvian **economic analyst** :bowtie:.
+
+* :school_satchel: Education: [*Harris School of Public Policy*](https://harris.uchicago.edu/)
+* :bank: Work: [*Central Reserve Bank of Peru*](https://www.bcrp.gob.pe/en)
+* Contact information:
```

Comment:
Maybe you can separate your contact information from education and work by making it a new header or just make it the same format as the first introduction.

Reply:
Or maybe you can add an emoji before contact information to make it the format as education and work.